typhos/positioner.py

```diff
@@ -476,25 +476,30 @@ def _define_setpoint_widget(self):
             self.ui.set_value.addItems(setpoint_signal.enum_strs)
             # Activated signal triggers only when the user selects an option
             self.ui.set_value.activated.connect(self.set)
-            self.ui.set_value.setSizePolicy(
-                QtWidgets.QSizePolicy.Expanding,
-                QtWidgets.QSizePolicy.Fixed,
-            )
             self.ui.set_value.setMinimumContentsLength(20)
             self.ui.tweak_widget.setVisible(False)
         else:
             self.ui.set_value = QtWidgets.QLineEdit()
             self.ui.set_value.setAlignment(QtCore.Qt.AlignCenter)
             self.ui.set_value.returnPressed.connect(self.set)
 
+        self.ui.set_value.setSizePolicy(
+            QtWidgets.QSizePolicy.Fixed,
+            QtWidgets.QSizePolicy.Fixed,
+        )
+        self.ui.set_value.setMinimumWidth(
+            self.ui.user_setpoint.minimumWidth()
+        )
         self.ui.set_value.setMaximumWidth(
             self.ui.user_setpoint.maximumWidth()
         )
-
         self.ui.setpoint_layout.addWidget(
             self.ui.set_value,
             alignment=QtCore.Qt.AlignHCenter,
         )
+        self.ui.set_value.setObjectName('set_value')
```

Comment:
I know Ken was doing this for testing, but should we be doing this more uniformly in our UI's?  What does setting the object name here do?

Reply:
There are two things this does:
1. The widget now has a name when inspected via gammaray. This is the only reason I did this here, technically I could remove it. I suspect this could also be helpful for tracebacks/other debug like Ken was doing for testing.
2. There are some qt functions for collecting child widgets based on their name: https://doc.qt.io/qt-6/qobject.html#findChild, and they use this qt object name rather than the python attribute name

Reply:
Additionally, it can be used as a method of differentiating widgets in the stylesheet